agent/bench-scripts/pbench_fio

```diff
@@ -304,24 +304,32 @@ function fio_process_options() {
 
 # Ensure the right version of the benchmark is installed
 function fio_install() {
-	if [ "$postprocess_only" == "n" ]; then
+	if [ -e $benchmark_bin ]; then
+		debug_log "[$script_name]$benchmark is already installed"
+		echo "$benchmark is already installed" 
+	elif [ "$postprocess_only" == "n" ]; then
 		if check_install_rpm $benchmark $ver; then
 			debug_log "[$script_name]$benchmark is installed"
 		else
 			debug_log "[$script_name]$benchmark installation failed, exiting"
 			exit 1
-		fi
-		if [ ! -z "$clients" ]; then
-			debug_log "verifying clients have fio installed"
-			echo "verifying clients have fio installed"
-			for client in `echo $clients | sed -e s/,/" "/g`; do
-				ssh $ssh_opts $client ${pbench_install_dir}/bench-scripts/pbench_fio --install &
-			done
-			wait
-		fi
-		if [ "$install_only" == "y" ]; then
-			exit 0
-		fi
+		fi 
+	fi
+	if [ ! -z "$clients" ]; then
+		debug_log "verifying clients have fio installed"
+		echo "verifying clients have fio installed"
+		for client in `echo $clients | sed -e s/,/" "/g`; do
+			if ( ssh $ssh_opts $clients "[ ! -e $benchmark_bin ]" ); then 
+				ssh $ssh_opts $client yum -y install $benchmark-$ver
```

Comment:
@ndokos we have only yum at this place in pbench_fio and with state as is now, it is not assumed pbench packages are installed on client systems ( where actual test run ). If there is requirement to use here check_installed_rpm, then we must have pbench tools installed at other side - client system in order to get rid of this yum.What do you think?